src/static/css/infusion-docs.css

```diff
@@ -73,6 +73,24 @@ a:focus {
     outline: 0.2rem solid #ffb200;
 }
 
+#skip a,
+#skip a:hover,
+#skip a:visited {
+    height: 1px;
+    left: -10000px;
+    overflow: hidden;
+    position: absolute;
+    top: auto;
+    width: 1px;
+}
+
+#skip a:active,
+#skip a:focus {
+    height: auto;
+    position: static;
+    width: auto;
```

Comment:
When the links appear on focus, the whole page shifts down. I wonder if it's possible to do something with the styling so that the links appear over top of the header?